src/app/app.component.html

```diff
@@ -67,10 +67,6 @@
 	</nav>
 </header>
 
-<div class="spinner-container" *ngIf="logout">
```

Comment:
This whole block, is now not necessary since we won't have a logout action anymore